composer.json

```diff
@@ -11,8 +11,9 @@
         "webmozart/assert": "^1.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.6",
-        "php-http/guzzle6-adapter": "^1.0"
+        "phpunit/phpunit": "~4.8",
+        "php-http/guzzle6-adapter": "^1.0",
+        "phpunit/php-code-coverage": "^2.2.4"
```

Comment:
I thought that PHPUnit was offering "by default" a way to build a code coverage report, even if that is not the case, a code coverage tool - IMHO - should be a local vendor and never thought about it as "the panacea" given some cases may not cover 100% and we really want to do that. given my experience, the only way to validate our code - 100% - is writing tests **before** to add a single line of code. I don't know if is a good idea to add this vendor, even for a `dev` environment... anyway, this is just my opinion...